backend/src/models/CampusTutorModel.ts

```
@@ -0,0 +1,231 @@
+  }
+
+  public static getBookings(): BookingSession[] {
+    return [...this.bookings];
+  }
+
+  public static bookSession(
+    tutorId: string,
+    studentName: string,
+    scheduledTime: string,
+    durationMinutes: number,
+    sessionType: 'one-on-one' | 'group' | 'exam-prep'
```

ESLint

[error] Replace `'one-on-one'·|·'group'·|·'exam-prep'` with `"one-on-one"·|·"group"·|·"exam-prep",`

(prettier/prettier)